src/components/PageNavigator/PageNavigatorTs.ts

```diff
@@ -41,4 +42,8 @@ export class PageNavigatorTs extends Vue {
     await this.$store.dispatch('account/LOG_OUT')
     this.$router.push({name: 'accounts.login'})
   }
+  
+  public onPageNavigate(route: Route) {
+    this.currentAccount && this.$router.push({ name: route.name }).catch(()=>{/**/})
```

Comment:
This error should not be obfuscated. If it happens, it means that likely the **button** is not being disabled when the page is active.

Please, do not obfuscate the error, make sure it is _logged_, or that _it catches only this error._

An example exploit here would be to inject `hello-world` inside `route.name`. The router will throw an unknown route exception but your code will obfuscate it.